engine/src/w32-clipboard.cpp

```diff
@@ -1246,6 +1247,14 @@ MCDataRef MCWin32RawClipboardItemRep::CopyData() const
     }
     
 	// Update the data cache and return it
+	if (t_data == nullptr)
+	{
+		MCExecContext ctxt(MCdefaultstackptr, nullptr, nullptr);
```

Comment:
@livecodealex this doesn't seem right for a couple of reasons:
- we generally wouldn't throw an error for something like this that is not really in the user's control. We might set the result to an error string but I guess that's not normal with global property access either...
- more importantly though this method returns null elsewhere so I'm not sure why it isn't safe to return null here

Reply:
Yeah fair enough I guess, I am currently working on a patch to make it actually return something.

Reply:
Though we still return null, it's just that I think it's better to throw errors than to have undefined behaviour. Either way, the 

```
if (t_data == nullptr)
{
}
```

is still necessary, as that allows us to avoid a crash.